metapool/metapool.py

```diff
@@ -18,6 +18,13 @@
                       'iSeq', 'NovaSeq']
 OTHER_SEQUENCERS = ['HiSeq2500', 'HiSeq1500', 'MiSeq']
 
+SYNDNA_POOL_NUM_KEY = 'syndna_pool_number'
```

Comment:
Maybe add comment above that specifies that these KEYS are the expected column headers from plate_df ?